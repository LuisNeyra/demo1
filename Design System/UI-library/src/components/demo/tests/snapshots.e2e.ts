import { snapIt } from '../../../test';

const component = 'demo';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('renders correctly with no props', '<blaze-demo></blaze-demo>');
    snap(
      'renders a badge component and a code area',
      `<blaze-demo language="language-html" code='<blaze-badge type="success">YO!</blaze-badge>'></blaze-demo>`
    );
  });
});
