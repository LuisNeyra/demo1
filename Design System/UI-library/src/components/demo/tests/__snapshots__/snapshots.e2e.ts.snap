// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`demo renders renders a badge component and a code area 1`] = `"<blaze-demo language=\\"language-html\\" code=\\"<blaze-badge type=&quot;success&quot;>YO!</blaze-badge>\\" class=\\"hydrated\\"><div aria-label=\\"Demo component\\" class=\\"demo-markup-source u-letter-box-small\\"><blaze-badge type=\\"success\\" class=\\"hydrated\\"><!----><span class=\\"c-badge c-badge--success\\">YO!</span></blaze-badge></div><div class=\\"u-letter-box-small\\"><pre class=\\"u-pre\\"><code aria-hidden=\\"\\" tabindex=\\"-1\\" class=\\"u-code u-code--multiline language-html\\" contenteditable=\\"\\">&lt;blaze-badge type=\\"success\\"&gt;YO!&lt;/blaze-badge&gt;</code></pre></div></blaze-demo>"`;

exports[`demo renders renders correctly with no props 1`] = `"<blaze-demo class=\\"hydrated\\"><div aria-label=\\"Demo component\\" class=\\"demo-markup-source u-letter-box-small\\"></div><div class=\\"u-letter-box-small\\"><pre class=\\"u-pre\\"><code aria-hidden=\\"\\" tabindex=\\"-1\\" class=\\"u-code u-code--multiline\\" contenteditable=\\"\\"></code></pre></div></blaze-demo>"`;
